armsrc/lfops.c

```diff
@@ -1417,6 +1417,56 @@ void WriteT55xx(uint32_t *blockdata, uint8_t startblock, uint8_t numblocks) {
 	}
 }
 
+// Copy Paradox ID to card and setup block 0 config
+void CopyParadoxtoT55x7(uint32_t hi2, uint32_t hi, uint32_t lo, uint8_t longFMT) {
+	uint32_t data[] = {0,0,0,0,0,0,0};
+	uint8_t last_block = 0;
+
+	if (longFMT) {
```

Comment:
I'm not sure, and I don't think there's an easy way to find out. At the very least, by having support for it here, we can cover both cases and it should work regardless of the code is never actually used.

Reply:
i don't believe there is a long format for paradox.  but if we can refactor a little so as to not duplicate the code it will do no harm. (will comment on the refactor next...)